region/brinstar/kraid.json

```diff
@@ -51,7 +51,21 @@
           "name": "Elevator to Business Center",
           "nodeType": "door",
           "nodeSubType": "elevator",
-          "nodeAddress": "0x0019246"
+          "nodeAddress": "0x0019246",
+          "canLeaveCharged": [
+            {
+              "framesRemaining": 130,
+              "usedTiles": 17,
+              "strats": [
+                {
+                  "name": "Base",
+                  "notable": false,
+                  "requires": [{"obstaclesCleared": ["A"]}]
```

Comment:
could have another strat without the obstacle being broken, with `canComeInCharged`